forge-gui/res/cardsfolder/upcoming/the_master_formed_anew.txt

```diff
@@ -0,0 +1,9 @@
+Name:The Master, Formed Anew
+ManaCost:U B
+Types:Legendary Creature Time Lord Rogue
+PT:0/1
+K:ETBReplacement:Copy:DBCopy:Optional
+T:Mode$ SpellCast | ValidCard$ Card.Self | Execute$ TrigExile | OptionalDecider$ You | TriggerDescription$ Body Thief — When you cast this spell, you may exile a creature you control and put a takeover counter on it.
+SVar:TrigExile:DB$ ChangeZone | Origin$ Battlefield | Destination$ Exile | Hidden$ True | ChangeType$ Creature.YouCtrl | ExileWithCounter$ TAKEOVER
+SVar:DBCopy:DB$ Clone | Choices$ Creature.counters_GE1_TKVR | ChoiceZone$ Exile | SpellDescription$ You may have CARDNAME enter the battlefield as a copy of a creature card in exile with a takeover counter on it.
```

Comment:
The only spot that uses the abbreviated counter name is CounterEnumType.java – the abbreviation is a GUI thing so when the board state gets cluttered we don't have long counter names covering multiple cards